config/routes.rb

```diff
@@ -1,4 +1,32 @@
 Rails.application.routes.draw do
+
+  root :to => 'main#index'
+
+  get '/login', :to => 'sessions#new', :as => 'login'
+  post '/login', :to => 'sessions#create'
+  delete '/logout', :to => 'sessions#destroy'
+  # get '/dashboard', :to => 'users#show', :as => 'dashboard'
+
+  resources :users, only: [:new, :create]
+
+  resources :dashboard, only: [:index]
+
+  # namespace :dashboard do
+  #   resources :user, only: [:show]
+  # end
+
+  # scope(path_names: { show: ('dashboard')}) do
+  #   resources :users, path: 'dashboard', only: :show
+  # end
+  #
+  # resources :users, only: [:show] do
+  #   collection do
+  #     get :dashboard
+  #   end
+  # end
+
```

Comment:
Let's remove these comments before we merge this branch into master.

Reply:
Will remove into a waffle issue so that we can keep track of the fact that slugging has not actually been implemented. @mimilettd @samuelssnider @JunePaloma